configure.ac

```diff
@@ -137,7 +137,7 @@ AC_MSG_CHECKING([for jar installation path])
 AC_ARG_WITH(jardir,
             AS_HELP_STRING([--with-jardir],
                            [jar file installation path [LIBDIR/jni]]),
-            jardir=${withval}, jardir=${libdir}/jni)
+            jardir=${withval}, jardir=${prefix}/lib/jni)
```

Comment:
The default value of ${libdir} should already be ${prefix}/lib (set by autoconf). Are you sure this is needed?